frontend/src/routes/Login.js

```diff
@@ -16,13 +19,28 @@ class Login extends Component {
 
     const logo = {
       width: "30%",
-      paddingTop: "40px"
+      paddingTop: "2.5rem"
+    };
+
+    const bluePurpleGradient = {
+      background: "linear-gradient(45deg, #721ebe, #3260b3)",
+      borderRadius: "0rem 5rem 0rem 0rem",
+      marginLeft: "-0.9375rem",
+      marginTop: "1.25rem",
+      height: '97vh',
+      paddingTop: "5rem"
     };
 
+
+    const paddingTop = {
```

Comment:
this name tells me nothing of what this class is for (like what elements)

Reply:
My bad I'll fix it!

Reply:
Changed to `signinSpacer`.  It is just a spacer so the Sign In text starts after the curve of the gradient on the left side.  I don't know a better name for this so I think it's good enough.